### DISCUSSION TITLE
KingArt Dimmer Query

### DISCUSSION THREAD

**Discussion by muzzthegreat** [arendst/Tasmota#12655] [Q&A] [answered]
Hi all, 
I recently bought a KingArt King-M4 Dimmer ([something like this](https://www.aliexpress.com/item/32891383747.html?spm=a2g0s.9042311.0.0.300d4c4dx6emz1)). I have managed to flash the device with Tasmota (https://tasmota.github.io/docs/devices/KingArts-touch-Dimmer/)  and set the device module type to PS-16-DZ(58).

I've picked up the device in TasmoAdmin and Home Assistance with no problems.

It seems to work. The lights turn on and off, but the dimmer is very erratic. It seems to jump when holding down the down dimmer, then the lights start flickering, some of the lights on the circuit turn off, then turn back on when holding the up dimmer button.

I checked the voltage on the input (230V) but the output voltage at max brightness is measuring about 180V and then drops down to about 150V when dimming. I'm not surprised that the lights don't work correctly. 

I'm wondering if the output doesn't know what 100% brightness is or what 10% is. Maybe what it thinks is 100%, is actually 40% and why the lights aren't working.

I installed the exact same dimmer on another circuit using the native firmware (controlled through ewelink) and this seems to work correctly, although having said that, it does flicker the lights when it hits 10% (which could be understandable).

Any way to check and recalibrate the brightness or why the output voltage is so low?

Thanks

**Comment by sfromis** [comment, marked as answer]
If you use LED or CFL lights behind a dimmer, you're just *asking* for trouble. Don't expect some multimeter to give a meaningful voltage on the output side of a dimmer.

Do verify correct and solidly attached wiring.
